shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -584,33 +575,19 @@ static inline PointerChangeMapperPhase PointerChangePhaseFromUITouchPhase(UITouc
   return blink::PointerData::DeviceKind::kTouch;
 }
 
-- (void)dispatchTouches:(NSSet*)touches phase:(UITouchPhase)phase {
-  // Note: we cannot rely on touch.phase, since in some cases, e.g.,
-  // handleStatusBarTouches, we synthesize touches from existing events.
-  //
-  // TODO(cbracken) consider creating out own class with the touch fields we
-  // need.
-  auto eventTypePhase = PointerChangePhaseFromUITouchPhase(phase);
+// Dispatches the UITouches to the engine. Usually, the type of change of the touch is determined
+// from the UITouch's phase. However, the AppDelegate fakes touches to simulate status bar touches
+// when the Flutter view is no full screen. The change type of the faked touch is specified in the
+// second argument. If the `overridden_change` is nullptr, the change type is determined from
```

Comment:
'change' seems like a non-term-of-art wording. Should we call it phase_override or something?

Reply:
I was using the PointerData::Change enum as the basis of that wording.